protostar/commands/test/test_command.py

```diff
@@ -152,6 +152,16 @@ def arguments(self):
                 type="bool",
                 description="Only re-run failed and broken test cases.",
             ),
+            ProtostarArgument(
+                name="gas-price",
+                type="float",
+                description=(
+                    "Gas price per gas unit. If provided, Protostar will estimate a fee for test cases. "
```

Comment:
I would add disclamer that those estimations may be heavily inaccurate

Reply:
If estimations may be heavily inaccurate, then this feature is pointless.

Reply:
Why?

Reply:
StarkNet estimations may be inaccurrate as well, it is an estimation

Reply:
Estimations are inaccurate by definition, hence a disclaimer isn't needed.

Reply:
I think It wouldnt hurt to avoid any misunderstandings

Reply:
[update description](https://github.com/software-mansion/protostar/pull/1195/commits/4c2325600ff2eea98710ae1fdfdff612e73408c5)